docs/data-guidelines/index.md

```diff
@@ -36,6 +36,10 @@ For Safari for iOS, use the iOS version number, not the Safari version number or
 
 This versioning scheme came at [Apple's request, in #2006](https://github.com/mdn/browser-compat-data/issues/2006#issuecomment-457277312).
 
+### Bun versioning
+
+Since Bun upgrades the WebKit version in every patch release, which may include new features, BCD tracks patch releases for Bun.
```

Comment:
Slight semantics nit on the wording here: While it's true that we do upgrade WebKit versions in patches, it's not _always_ the case that a patch will have a new version.

Reply:
```suggestion
Since Bun upgrades the WebKit version also in patch releases, which may include new features, BCD tracks patch releases for Bun.
```